alacritty/src/input.rs

```diff
@@ -674,6 +708,32 @@ impl<T: EventListener, A: ActionContext<T>> Processor<T, A> {
         }
     }
 
+    fn on_message_bar_click(&mut self) {
```

Comment:
The function name should describe what this method actually does. It's not a click handler for the message bar.

Reply:
Right, it's a click to the box in the message bar. Appropriately renamed.